hbw/config/defaults_and_groups.py

```diff
@@ -277,6 +281,8 @@ def set_config_defaults_and_groups(config_inst):
         "SR": ("ggHH_kl_1_kt_1_sl_hbbhww", "qqHH_CV_1_C2V_1_kl_1_sl_hbbhww"),
         "vbfSR": ("ggHH_kl_1_kt_1_sl_hbbhww", "qqHH_CV_1_C2V_1_kl_1_sl_hbbhww"),
         "BR": lambda proc_name: "hbbhww" not in proc_name,
+        # "SR_dl": ("ggHH_kl_5_kt_1_dl_hbbhww"),
```

Comment:
To make a tuple of length 1, you need to add a comma. Otherwise the brackets are basically ignored by python
```suggestion
        # "SR_dl": ("ggHH_kl_5_kt_1_dl_hbbhww",),
```